events/resumed.js

```diff
@@ -0,0 +1,19 @@
+const { Event } = require("klasa");
+const { WebhookClient } = require("discord.js");
+
+module.exports = class extends Event {
+
+    constructor(...args) {
+        super(...args, { once: false });
+        this.webhook = new WebhookClient("451318929814716426", this.client.config.webhooks.status);
```

Comment:
if you're going to use this in *most* events you should probably just attach it to client instead of creating a new webhookClient multiple times